app/src/main/java/com/willowtree/vocable/presets/CategoriesFragment.kt

```diff
@@ -56,7 +56,7 @@ class CategoriesFragment : BaseFragment<CategoriesFragmentBinding>() {
         categories?.forEachIndexed { index, category ->
             val categoryButton =
                 CategoryButtonBinding.inflate(inflater, binding.categoryButtonContainer, false)
-            with(categoryButton.root as CategoryButton) {
+            with(categoryButton.root) {
```

Comment:
this is a good change bc we don't need it and also it's already named categoryButton :3 👍🏼